app/models/hanuman/observation.rb

```diff
@@ -121,6 +122,14 @@ def fill_answer
       end
     end
 
+    def edit_gnss
```

Comment:
Name could be more descriptive (which is important since other devs will just see it in the callback list and need to know what it means). Something like `check_location_metadata` would be good.